package.json

```diff
@@ -13,12 +13,15 @@
     "winston": "^2.1.1"
   },
   "devDependencies": {
+    "eslint": "^1.10.3",
+    "eslint-config-trails": "^0.0.1",
```

Comment:
This PR is dependent on publishing `eslint-config-trails`. Also, it might be your preference to remove `eslint` from the `devDependencies`